db/models.py

```diff
@@ -1 +1,29 @@
 from django.db import models
+from django.db.models import ForeignKey
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
```

Comment:
The `Race` model's `name` field correctly includes `unique=True`, ensuring that each race has a unique name, which aligns with the task requirements.